### PR TITLE
Working branch

### PR DESCRIPTION
**Pull request by edemagbenyo**

### CODE REVIEW

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Hi team :wave: nice work,

### However, there are some additions we need to make :smiley: 

- [ ] There are no linter checks on this project, also, you have to add .stickler.yml file to the root directory of this project and [use this information to configure your stickler.yml ](https://github.com/microverseinc/linters-config#linters-config). Go to [https://stickler-ci.com/](https://stickler-ci.com/) to sign up on stickler if you have not done so.

- [ ] Your README.md file is not detailed. Make it more descriptive, you can use this blue print.

     * A definition of this project
     * The authors of the project
     * Link to the **Clone**, you can use [Githack](https://raw.githack.com/) to get a link, by pasting the link of your html file from github on the field at **githack**
     * Also, you can provide a link to **Odin's project** for detailed information.


- [ ] There should be least three embedded images on the rightmost part of the navbar that represent icons and/or an avatar, you have only.

- [ ] Below the Video section together with **like and dislike icons and share button**  there should be **an avatar** of the user who uploaded the video, you missed that.

### After implememting these changes, kindly submit another form for code review.

### This project is not approved :stop_sign:

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @edemagbenyo and @collinsugwu  🌟 

**The project looks nice, however we need to implement some changes** ⬇️ 

1. The video is not well implemented on the page it must meet the following requirements: 
    - It is displayed
    -  It shows the video title
    -  It shows the number of views
2. Fix the number of views under the video.
3. Validate and fix the HTML markup: https://validator.w3.org/#validate_by_uri
4. Use Githack to create a live link for your project and put it in the README file after making the changes: https://raw.githack.com/

### _After finishing all the above changes please resubmit a code review request._

### Good luck guys! 🤞

**Review by wathigo — CHANGES_REQUESTED** [review]
## HI devs :wave: 
**Great work so far,** 
Before we give this a pass there are a few things you need to fix.
1. You CSS file has one error. Upload your code to [w3 validator](https://validator.w3.org/#validate_by_uri) and fix it.
2. There should be at least three embedded images at the right most part of the navigation bar. You only have two. Please add at least one. Also make sure the icons are aligned with the search bar.

**After implementing this changes, submit another request for review.**

_Good luck :)_

File: index.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./css/styles.css">
+    <title>Youtube</title>
+</head>
+
+<body>
+    <header class="header">
+        <div class="logo">
+            <img class="menu" src="./images/menu.png">
```
Comment (bolah2009) [inline index.html:15]: #### `HINT`

Please, always use `alt` attribute for `` element. :bulb: 

This is a very important accessibility attribute.
**Examples of scenarios where users benefit from text alternatives for images** are listed below and could also be found [here](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)

- They have a very slow connection and are browsing with images disabled.
- They have a vision impairment and use text to speech software.
- They have a cognitive impairment and use text to speech software.
- They are using a text-only browser.
- They are listening to the page being read out by a voice Web browser.
- They have images disabled to save on download costs.
- They have problems loading images or the source of an image is wrong.


##### SOURCE: [W3C Recommendation (Embedded Content)](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,24 @@
...
+Here is how to setup this project on your machine.
+
+Open your terminal in your working folder and execute the following command
+
+>git clone https://github.com/edemagbenyo/images-videos-collins-edem
+
+
+Here is the link to the live version of this project	
+[here](https://rawcdn.githack.com/edemagbenyo/images-videos-collins-edem/8749c5c20fd665623673327e159868e8a6f34916/index.html)
+
```
Comment (bolah2009) [inline README.md:24]: #### `OPTIONAL`

Nice job adding a descriptive readme.

Make it even more professional by adding the assignment link (from Odin),  screenshots of the page and what was learned during the project.

**Review by bolah2009 — CHANGES_REQUESTED** [review]
# Awesome job Team :clap: 

### The [clone](https://raw.githack.com/edemagbenyo/images-videos-collins-edem/824118b8536ac544166fdba27601a0bf501c58e3/index.html) looks good to me, uses HTML Video Embeding effectively. :clap: 

There are however a few required changes  to make the project even better  :blush:  :
 - Kindly rename the PR (to the name of the milestone) and the PR description to a brief summary of what was done in this PR. (Examples Below)
- Fix [HTML validation errors/warnings](https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2Fedemagbenyo%2Fimages-videos-collins-edem%2F824118b8536ac544166fdba27601a0bf501c58e3%2Findex.html), check comments for more details.
  > Validate your HTML markup with https://validator.w3.org/#validate_by_upload.

Kindly, check the comment below for suggestions.

#### :recycle:  Make these changes and submit another code review request. :+1: 

#### Feel free to reach me on slack if you having issues implementing these features. :+1: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Review by bolah2009 — COMMENTED** [review]
# Rename PR

  rename PR

-----

# Update PR description

  update-PR-description-html

**Review by whiz25 — APPROVED** [review]
## Awesome! :clap: 
Congratulations. You met all requirements for this project.
Please go ahead and merge and I wish you success in your next projects.

**Pull request merged**